### DISCUSSION TITLE
When applying / cancelling coupon code the page refreshes twice

### DISCUSSION THREAD

**Discussion by pabodah** [breezefront/community#173] [5. General]
Hello,

**Issue**
1. When applying or cancelling coupon code in the cart page, the page refreshes twice.

Recreate the issue
1. Create a cart price rule in backend
2. Add a product to the cart
3. Go to cart page
4. Enter the coupon code and click on apply discount

[Screencast 2023-07-14 14-43-03.webm](https://github.com/breezefront/community/assets/1543867/4c6147fc-0deb-4958-8af8-50764cfee30c)

**Comment by vovayatsyuk** [comment]
Thank you for reporting!

Fixed: https://github.com/breezefront/module-breeze/commit/6eafee022b1e7d177d228cd31f0415d388d09c0d
